simplemind/models.py

```
@@ -46,6 +46,9 @@ class Conversation(SMBaseModel):
     def __str__(self):
         return f"<Conversation id={self.id!r}>"
 
+    def prepend_system_message(self, role: str, text: str, meta: Dict[str, Any] = {}):
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)